src/qml/ThreadDelegate.qml

```diff
@@ -24,13 +24,18 @@ import Ubuntu.Telephony 0.1
 import Ubuntu.Contacts 0.1
 import QtContacts 5.0
 import Ubuntu.History 0.1
+import QtGraphicalEffects 1.0
+
 import "dateUtils.js" as DateUtils
 
 ListItemWithActions {
     id: delegate
 
     property QtObject chatEntry: null
     property bool compactView: false
+    property bool isEmptyMMS: eventTextMessageType === HistoryEventModel.MessageTypeMultiPart && displayedEventTextAttachments.length === 0 && eventTextMessage.length === 0
+    property string defaultMMSErrorMessage: i18n.tr("Oops, there has been an error with the MMS system and this message could not be retrieved.
```

Comment:
Why is this needed here, isnt that the same as https://github.com/ubports/telephony-service/pull/13/files#diff-e65b0117bee632c68e29f9596763e0867a876da2ed407c71ab9c6752e54c7f77R504 ?

Reply:
yes the same, but translation can't be share between those two afaik ?

Reply:
No, translation cannot be shared. But if you use the exact same string the translation server can update it in both :)

Reply:
Oh, so tell me how to do it ?, translation will be exactly the same

Reply:
@Flohack74 can we go ahead for that PR ?